### ISSUE TITLE
wrong parsing arguments with space 

### ISSUE THREAD

**Issue by Junker** [afonsoc12/firefly-cli#14]
Problem with parsing arguments with space from cli:

```
$> firefly-cli add  --source-name "My Bank Account" --amount 499 --description "Food" --destination-name "Shop"
Transaction header:
+-----------------------+-------------------------+
| header__apply_rules   | header__fire_webhooks   |
|-----------------------+-------------------------|
| True                  | False                   |
+-----------------------+-------------------------+

Transaction Body:
+--------------------------+----------+---------------+---------------+--------------------+------------+
| date                     |   amount | description   | source_name   | destination_name   | type       |
|--------------------------+----------+---------------+---------------+--------------------+------------|
| Thu, 2023-04-06 19:15:25 |      499 | Food          | My            | Shop               | withdrawal |
+--------------------------+----------+---------------+---------------+--------------------+------------+
```

As you can see, the source name is **"My"** instead of **"My Bank Account"**.
But if i do it from tui mode, it works fine:

```
$> firefly-cli                                                                                                                              

Copyright 2023 Afonso Costa

Licensed under the Apache License, Version 2.0 (the "License");
Type "license" for more information.

Welcome to FireflyIII Command Line Interface!
Created by Afonso Costa (@afonsoc12)

=============== Status ===============
  - URL: https://***********
  - API Token: *****
  - Version: v0.1.2
  - Connection: OK!
======================================

Type "help" to list commands.

🐷 ➜ add --source-name "My Bank Account" --amount 499 --description "Food" --destination-name "Shop" 
Transaction header:
+-----------------------+-------------------------+
| header__apply_rules   | header__fire_webhooks   |
|-----------------------+-------------------------|
| True                  | False                   |
+-----------------------+-------------------------+

Transaction Body:
+--------------------------+----------+---------------+-----------------+--------------------+------------+
| date                     |   amount | description   | source_name     | destination_name   | type       |
|--------------------------+----------+---------------+-----------------+--------------------+------------|
| Thu, 2023-04-06 19:15:53 |      499 | Food          | My Bank Account | Shop               | withdrawal |
+--------------------------+----------+---------------+-----------------+--------------------+------------+

Would you like to proceed adding the transaction? (y/n): 
```

**Comment by afonsoc12** [comment]
Hi @Junker, 
Thanks for reporting this bug, I'll investigate and let you know more in a couple of days

**Comment by afonsoc12** [comment]
The issue was that after splitting cmd arguments from generic ones (like --help and --version), it was not considering whitespaced strings as a single argument.
Should be fixed on v0.2.1, currently deploying.

Let me know if you have any issues, will close it for now 👍
